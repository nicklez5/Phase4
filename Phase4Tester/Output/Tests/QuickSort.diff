--- SelfTestCases/QuickSort.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/QuickSort.out	2020-03-14 21:50:47.122514388 -0700
@@ -9,15 +9,15 @@
 19
 5
 9999
-2
-5
 6
-7
-9
 11
-12
-18
+9
 19
+5
+7
 20
+2
+12
+18
 0
 
