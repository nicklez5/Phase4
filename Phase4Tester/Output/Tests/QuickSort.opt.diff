--- SelfTestCases/QuickSort.opt.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/QuickSort.opt.out	2020-03-14 13:04:38.053787052 -0700
@@ -1,23 +1,12 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-9999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 372 column 11: "0": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 386 column 11: "1": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 400 column 11: "2": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 414 column 11: "3": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 428 column 11: "4": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 442 column 11: "5": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 456 column 11: "6": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 470 column 11: "7": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 484 column 11: "8": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 498 column 11: "9": operand is of incorrect type
 
+Processing terminated due to errors.
