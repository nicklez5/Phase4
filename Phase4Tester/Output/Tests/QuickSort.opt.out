Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 372 column 11: "0": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 386 column 11: "1": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 400 column 11: "2": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 414 column 11: "3": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 428 column 11: "4": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 442 column 11: "5": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 456 column 11: "6": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 470 column 11: "7": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 484 column 11: "8": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/QuickSort.opt.compiler.out line 498 column 11: "9": operand is of incorrect type

Processing terminated due to errors.
